go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/operator-framework/operator-sdk v0.17.0
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/client_model v0.2.0
-	github.com/rook/rook v1.4.3-0.20200901124914-0b665e6e3315
+	github.com/rook/rook v1.4.6
```

Comment:
Is this used in the downstream builds? v1.4.6 has diverged a bit from what we're using downstream. Some fixes have been backported to the 1.4 branch upstream, but not to the downstream 4.6 branch. OCS should really be referencing the downstream rook branch under github.com/openshift/rook, but it's really a separate topic than for this PR.